source

```diff
@@ -101775,11 +101775,13 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    character's code point) to the current tag token's tag name.</dd>
 
    <dt>U+0000 NULL</dt>
-   <dd><span>Parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current tag
+   <dd>This is an <span data-x="parse-error-unexpected-null-character">unexpected-null-character</span>
+   <span>parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current tag
    token's tag name.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd>This is an <dfn data-x="parse-error-unexpected-eof-character">unexpected-eof-character</dfn>
```

Comment:
Unlike for `null` characters which are treated nearly the same way in all cases and error semantics have same motivation, `eof` behave differently in different cases and not always treated as errors. Therefore, it makes sense to carefully examine context of `EOF`. E.g. here the cause of error could be forgotten `>`, but if `EOF` met e.g. in `<script>` it's not an error, because `</script>` at the end of the file is not mandatory. In this particular case let's name it `eof-in-tag-name`.